wake-lock/wakelock-object-is-independent.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WakeLock object is independent</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/wake-lock/">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+
+<script>
+
+function make_script() {
+  return '<script>' +
+         '  let iframeWakeLock, iframeRequest;' +
+         '  window.onmessage = message => {' +
+         '    if (message.data === "INIT") {' +
+         '      navigator.getWakeLock("screen")' +
+         '      .then(wakeLock => {' +
+         '        iframeWakeLock = wakeLock;' +
+         '        parent.postMessage(iframeWakeLock.active, "*");' +
+         '      });' +
+         '    } else if (message.data === "ACQUIRED") {' +
+         '      iframeRequest = iframeWakeLock.createRequest();' +
+         '      parent.postMessage(iframeWakeLock.active, "*");' +
+         '    } else if (message.data === "RELEASED") {' +
+         '      iframeRequest.cancel();' +
+         '      parent.postMessage(iframeWakeLock.active, "*");' +
+         '    }' +
+         '  };' +
+         '<\/script>';
+}
+
+function load_iframe(src) {
+  return new Promise(resolve => {
+    const iframe = document.createElement("iframe");
+    iframe.onload = () => { resolve(iframe); };
+    iframe.srcdoc = src;
+    iframe.style.display = "none";
+    document.documentElement.appendChild(iframe);
```

Comment:
`document.body.appendChild()`... adding it to the HTML element doesn't make sense.